cld-video-player.css module not found

> @demptd13 @victorcodess with this release, you can now import the player stylesheet from the next-cloudinary package which should make the player work properly in the App Router
> 
> ```
> import 'next-cloudinary/dist/cld-video-player.css';
> ```
> 
> https://next.cloudinary.dev/cldvideoplayer/basic-usage
> 
> please let me know if you experience any issues!

Hi there, thank you for your hard work!  When importing the css file I'm getting an error
Module not found: Package path ./dist/cld-video-player.css is not exported from package

_Originally posted by @coderdix24 in https://github.com/colbyfayock/next-cloudinary/issues/248#issuecomment-1858931938_

--- a/next-cloudinary/package.json
+++ b/next-cloudinary/package.json
@@ -15,6 +15,7 @@
       "import": "./dist/helpers.mjs",
       "require": "./dist/helpers.js"
     },
+    "./dist/cld-video-player.css": "./dist/cld-video-player.css",
     "./package.json": "./package.json"
   },
   "scripts": {

